jenkins-builds/1.4.0-21.09-eiger

```diff
@@ -9,6 +9,7 @@
  GSL-2.7-cpeAMD-21.09.eb
  GSL-2.7-cpeCray-21.09.eb
  ParaView-5.9.1-cpeCray-21.09-OSMesa-python3.eb
+ ParaView-5.10.0-cpeCray-21.09-OSMesa.eb --set-default-module
```

Comment:
```suggestion
 ParaView-5.10.0-cpeCray-21.09-OSMesa.eb
```

Reply:
Thanks @jfavre: I have suggested to remove the old version from the production list, since the module won't be removed. Anyway, the flag to update the default is not needed, as Lmod takes the last module as default

Reply:
Thanks @lucamar for your tips